src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -15,35 +16,43 @@
 public class ClimbSubsystem extends SubsystemBase {
   // Put methods for controlling this subsystem
   // here. Call these from Commands.
-
+  private Joystick m_climbJoystick;
   private MotorController m_climbMotorControllerOne;
   private MotorController m_climbMotorControllerTwo;
   private DigitalInput m_limitSwitch;
+  private double climbHight;
+  private boolean toggleClimb;
 
   // TODO: maybe add servos
 
-  public ClimbSubsystem() {
+  public ClimbSubsystem(Joystick joystick) {
+    m_climbJoystick = joystick;
+    toggleClimb = false;
```

Comment:
toggleClimb doesn't seem to do anything, just set to False here, and toggled in the method.  

Should it be used for some logic to prevent climb, or prevent the command from being scheduled?

Also would be good to have this displayed on shuffleboard at least in a debugging tab.